packages/shared/lib/contexts/implicit-account-creation/enums/account-state.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ImplicitAccountCreationState {
+    ONE_TIME_DEPOSIT = 'OneTimeDeposit',
+    FUND_CONFIRMATION = 'FundConfirmation',
+    ACCOUNT_CREATION = 'AccountCreation',
```

Comment:
We should not make these in all caps and snake, it should be like `PascalFormat`